Description: lintian reports bad whatis entries in manpages. Fixing here
Author: Paul Gevers <elbrus@debian.org>
Forwarded: http://bugs.motifzone.net/show_bug.cgi?id=1590
Last-reviewed: 17 Feb 2013

--- a/doc/man/man3/XmColorSelector.3
+++ b/doc/man/man3/XmColorSelector.3
@@ -1,7 +1,7 @@
 .DT
 .TH XmColorSelector 3X ""
 .SH NAME
-The Color Selector widget
+XmColorSelector \- The Color Selector widget
 .SH SYNOPSIS
 #include <Xm/ColorS.h>
 .SH DESCRIPTION
--- a/doc/man/man3/XmExt18List.3
+++ b/doc/man/man3/XmExt18List.3
@@ -1,7 +1,7 @@
 .DT
 .TH XxExt18LIst 3x ""
 .SH NAME
-The Internationalized Extended List widget
+XmExt18List \- The Internationalized Extended List widget
 .SH SYNOPSIS
 #include <Xm/Ext18List.h>
 .SH DESCRIPTION
--- a/doc/man/man3/XmFontSelector.3
+++ b/doc/man/man3/XmFontSelector.3
@@ -1,7 +1,7 @@
 .DT
 .TH XmFontSelector 3X ""
 .SH NAME
-The Font Selector widget
+XmFontSelector \- The Font Selector widget
 .SH SYNOPSIS
 #include <Xm/FontS.h>
 .SH DESCRIPTION
--- a/doc/man/man3/XmIconButton.3
+++ b/doc/man/man3/XmIconButton.3
@@ -1,7 +1,7 @@
 .DT
 .TH XmIconButton 3X "15 July 1992"
 .SH NAME
-The Icon Button widget
+XmIconButton \- The Icon Button widget
 .SH SYNOPSIS
 Documents the XmIconButton widget
 .SH DESCRIPTION
